consensus/src/network_tests.rs

```diff
@@ -34,6 +34,15 @@ use std::{
 };
 use tokio::runtime::Handle;
 
+#[derive(Clone)]
+pub struct NodeId {
```

Comment:
I'd recommend using `TwinsId` and some comments for this struct

Reply:
Done.